qt/scientific_interfaces/Indirect/test/ConvFitDataPresenterTest.h

```diff
@@ -46,13 +46,67 @@ class MockConvFitDataView : public IIndirectFitDataView {
   /// Public Methods
   MOCK_CONST_METHOD0(getDataTable, QTableWidget *());
   MOCK_METHOD1(validate, UserInputValidator &(UserInputValidator &validator));
+  MOCK_METHOD2(addTableEntry, void(size_t row, FitDataRow newRow));
```

Comment:
Just a heads up, but not something you need to do, this is the old style Gmock decelerations. The newer ones are much easier to work with as you don't need to count params or check if it's a const type

```c++
// Interface.h
virtual void foo(int, double &) const = 0;

// Test file
MOCK_METHOD(void, foo, (int, double&), (const, override));
```